packages/storage/src/indexed-db/index.ts

```diff
@@ -254,4 +255,54 @@ export class IndexedDb implements IndexedDbInterface {
   async saveGasPrices(value: GasPrices): Promise<void> {
     await this.u.update({ table: 'GAS_PRICES', value, key: 'gas_prices' });
   }
+
+  async getNotesForVoting(
+    addressIndex: AddressIndex | undefined,
+    votable_at_height: bigint,
+  ): Promise<[]> {
+    const relevantAssets = [];
+
+    // Fetch all assets and filter out the relevant asset IDs
+    let assetCursor = await this.db.transaction('ASSETS').store.openCursor();
+
+    while (assetCursor) {
+      // Test denom with a regular expression
+      const regex = new RegExp('_delegation_.*');
```

Comment:
I know this is still a draft PR, but I would rename this to indicate what the regex is looking for. For example:
```suggestion
      const isDelegationAssetType = new RegExp('_delegation_.*');
```